css/style.css

```diff
@@ -0,0 +1,283 @@
+    top,
+    #8c8c8c 0%,
+    #666666 50%,
+    #7c7c7c 52%,
+    #a8a8a8 100%
+  );
+  background: -webkit-linear-gradient(
+    top,
+    #8c8c8c 0%,
+    #666666 50%,
+    #7c7c7c 52%,
+    #a8a8a8 100%
+  );
+  background: linear-gradient(
+    to bottom,
+    #8c8c8c 0%,
+    #666666 50%,
+    #7c7c7c 52%,
+    #a8a8a8 100%
+  );
+  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#8c8c8c', endColorstr='#a8a8a8',GradientType=0 );
+}
+
+nav ul {
+  margin: 0;
+  padding: 0;
+}
+
+nav ul li {
+  margin: 0;
+  padding: 13px 0;
+  width: 10.5%;
+  float: left;
+  text-align: center;
+  display: table-cell;
+  border-left: 1px solid #9b9b9b;
+  border-right: 1px solid #3f3f3f;
+}
+
+nav ul li:first-child {
+  border-left: none;
+  border-right: 1px solid #3f3f3f;
+}
+
+nav ul li:last-child {
+  border-right: none;
+  border-left: 1px solid #3f3f3f;
+}
+
+nav ul li:hover {
+  background: rgba(0, 0, 0, 0.5);
+}
+
+nav ul li:last-child:hover {
+  border-right: none;
+  border-left: 1px solid #3f3f3f;
+  background: none !important;
+}
+
+nav ul li a {
+  font-size: 14px;
+  color: #ffffff;
+}
+
+.list-inline li {
+  display: inline-block;
+}
+
+.grab-search {
+  float: left;
+  margin:0 4px;
+  border: 1px solid #383838 !important;
+  background: rgba(0, 0, 0, 0.5);
+  border-radius: 10px!important;
+}
+
+.grab-search input {
+  background: none !important;
+  border-radius: 5px;
+  padding:3px 0;
+  border: none;
+}
+
+a {
+  color: #567b95;
+  text-decoration: none;
+}
+
+.clear {
+  clear: both;
+}
+
+.bg-primary {
+  background: #567b95;
+}
+
+.text-primary {
+  color: #8697a3;
+}
+
+.no-margin {
+  margin: 0;
+}
+
+.no-padding {
+  padding: 0;
+}
+
+.bold {
+  font-weight: 700;
+}
+
+.no-bold {
+  font-weight: 100;
+}
+
+.image {
+  margin: 0;
+  padding: 0;
+}
+
+.image img {
+  width: 100%;
+  height: auto;
+}
+
+.container {
+  margin: 0 auto;
+  display: block;
+  width: 100%;
+  max-width: 1000px;
+}
+
+.left {
+  float: left;
+}
+
+.right {
+  float: right;
+}
+
+.text-left {
+  text-align: left;
+}
+
+.text-right {
+  text-align: right;
+}
+
+.row {
+  margin: 5px 0;
+  width: 100%;
+  clear: both;
+}
+
+.col-10 {
+  width: 9%;
+  padding: 0 0.5%;
+  float: left;
+}
+
+.col-20 {
+  width: 18%;
+  padding: 0 1%;
+  float: left;
+}
+
+.col-25 {
+  width: 25%;
+  float: left;
+}
+
+.push-20 {
+  margin-left: 18%;
+}
+
+.col-30 {
+  width: 28%;
+  padding: 0 1%;
+  float: left;
+}
+
+.col-40 {
+  width: 38%;
+  padding: 0 1%;
+  float: left;
+}
+
+.col-50 {
+  width: 48%;
+  padding: 0 1%;
+  float: left;
+}
+
+.col-60 {
+  width: 58%;
+  padding: 0 1%;
+  float: left;
+}
+
+.col-70 {
+  width: 68%;
+  padding: 0 1%;
+  float: left;
+}
+
+.col-80 {
+  width: 78%;
+  padding: 0 1%;
+  float: left;
+}
+
+.col-90 {
+  width: 88%;
+  padding: 0 1%;
+  float: left;
+}
+
+.welcome-section {
+  height: 80vh;
+  background-image: url('../images/navigator.jpg');
+  background-attachment: scroll;
+  background-size: initial;
```

Comment:
Change the `background-size` from `initial` to `cover` to make it always fill the whole screen.